### PR TITLE
Fix issue with sigmoid or identity normalization

### LINKED ISSUE #1: Typo in non-softmax normalizations
It looks like there is typo in https://github.com/thomlake/pytorch-attention/blob/31059b18fd222db9afb04e186a9e06dd203880af/attention/attention.py#L265-L266 . You probably want to use the compare `normalize` with the strings, not `w`. This is a fairly minor issue since attention pretty much always uses softmax normalization

### PR DESCRIPTION
**Pull request by DNGros**
Fixes #1. It looks like there is a typo with "w" instead of "normalized" which could cause an issue if using sigmoid or identity normalization. This PR fixes that typo, but doesn't actually test the correctness of these normalizations .

### CODE REVIEW

**Comment by thomlake** [comment]
Thanks for the PR @DNGros. I need to add better test around the less commonly used portions of this codebase someday.

**Comment by DNGros** [comment]
@thomlake no problem. Thanks for sharing and open sourcing your code! It was a big help to come across this repo with some cleanly written and tested code that could just be copied in.
